docs/spec.md

```diff
@@ -495,6 +498,26 @@ Unsubscribe from [connection graph updates](#connection-graph-update). Only supp
 }
 ```
 
+### Fetch asset
+
+Fetch an asset from the server. Only supported if the server previously declared that it has the `assets` [capability](#server-info).
+
+#### Fields
+
+- `op`: string `"fetchAsset"`
+- `assetId`: string, unique identifier to locate the asset
+- `requestId`: number, unique 32-bit unsigned integer which is to be included in the response
+
+#### Example
+
+```json
+{
+  "op": "fetchAsset",
+  "assetId": "file:///foo/bar.txt",
```

Comment:
I find it strange that in the studio PR this is the asset name yet here you are calling it the _id_.